int16 uav_id
uint8 question
uint8 FREE_DEPLOY_AREA=0
uint8 RESERVE_DEPLOY_AREA=1
---
uint8 answer
uint8 WAIT=0
uint8 OK=1
geometry_msgs/Point deploy_position
